app/schemas/plum_schema.rb

```diff
@@ -23,6 +23,7 @@ class PlumSchema < ActiveTriples::Schema
   property :series, predicate: RDF::URI("http://id.loc.gov/ontologies/bibframe/seriesStatement")
   property :coverage, predicate: RDF::Vocab::DC11.coverage
   property :date, predicate: RDF::Vocab::DC11.date
+  property :date_published, predicate: ::RDF::Vocab::DC.created # duplicate of date_created
```

Comment:
Use RDF::Vocab::DC.issued instead?
URI: 	http://purl.org/dc/terms/issued
Label: 	Date Issued
Definition: 	Date of formal issuance (e.g., publication) of the resource.